src/__tests__/extensions/replay/mutation-rate-limiter.test.ts

```diff
@@ -72,7 +71,7 @@ describe('MutationRateLimiter', () => {
 
     test('returns event if _any_ adds are left', () => {
         const event = makeEvent({
-            adds: [{ parentId: 0, nextId: 0, node: {} as unknown as serializedNodeWithId }],
```

Comment:
Will have to come back and type this once https://github.com/rrweb-io/rrweb/pull/1593 merges